apps/frontend/src/app/app.module.ts

```diff
@@ -8,17 +8,20 @@ import { NotFoundComponent } from './not-found/not-found.component';
 import { NxWelcomeComponent } from './nx-welcome.component';
 import { TournamentAdminComponent } from './tournament-admin/tournament-admin.component';
 import { HttpClientModule } from '@angular/common/http';
-
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { MaterialModule } from './material/material.module';
 @NgModule({
   declarations: [AppComponent, NotFoundComponent, NxWelcomeComponent, TournamentAdminComponent],
   imports: [
     CommonModule,
     BrowserModule,
     HttpClientModule,
+    MaterialModule,
     RouterModule.forRoot(
       routes,
       { enableTracing: true } // TODO: disable enableTracing again
-    )
+    ),
+    NoopAnimationsModule
```

Comment:
Please replace the import of `NoopAnimationsModule` with `BrowserAnimationsModule` as animations can really spice up the look of your application.